cmd/daemon/kubernetes/jobs.go

```diff
@@ -54,14 +62,30 @@ func (j JobInformer) handle(e interface{}) {
 			JobName:     job.Name,
 			Namespace:   job.Namespace,
 			Errors:      jobErrorMessages(job),
-			ArtifactID:  job.Annotations["lunarway.com/artifact-id"],
+			ArtifactID:  job.Annotations[artifactIDAnnotationKey],
 			AuthorEmail: job.Annotations["lunarway.com/author"],
```

Comment:
I believe it would be a good idea to extract this value to a const as well. It is set in multiple places and as the`artifactIDAnnotationKey` is also a const then we should align them.

WDYT?

Reply:
Yeah lets follow up with that. I only changed the other one as I duplicated its use with this change 🙌